mysql-test/columnstore/basic/r/mcs84_rollup.result

```diff
@@ -16,9 +16,35 @@ year	SUM(sales)
 SELECT year, SUM(sales) FROM booksales GROUP BY year WITH ROLLUP ORDER BY year;
 ERROR HY000: Incorrect usage of CUBE/ROLLUP and ORDER BY
 SELECT year, SUM(sales) FROM booksales GROUP BY year WITH ROLLUP;
-ERROR 42000: The storage engine for the table doesn't support MCS-1014: Rollup is currently not supported.
-SELECT year, SUM(sales) FROM booksales GROUP BY year ASC WITH ROLLUP;
-ERROR 42000: The storage engine for the table doesn't support MCS-1014: Rollup is currently not supported.
-SELECT year, SUM(sales) FROM booksales GROUP BY year DESC WITH ROLLUP;
-ERROR 42000: The storage engine for the table doesn't support MCS-1014: Rollup is currently not supported.
+year	SUM(sales)
+2014	173944
+2015	180518
+NULL	354462
+CREATE TABLE three_cols ( key1 INTEGER, key2 INTEGER, value INTEGER) ENGINE=COLUMNSTORE;
+INSERT INTO three_cols(key1, key2, value) VALUES
+(NULL, NULL, NULL)
+, (NULL, NULL,    1)
+, (NULL,    1,    2)
+, (   1,    1,    3)
+, (   1,    2,    4)
+, (   1,    2,    5)
+, (   2,    3,    6)
+, (   2,    3,    7);
+SELECT key1, key2, SUM(value), AVG(value), MIN(value), MAX(value), COUNT(value), COUNT(*) FROM three_cols GROUP BY key1, key2 WITH ROLLUP;
+key1	key2	SUM(value)	AVG(value)	MIN(value)	MAX(value)	COUNT(value)	COUNT(*)
+1	1	3	3.0000	3	3	1	1
+1	2	9	4.5000	4	5	2	2
+1	NULL	12	4.0000	3	5	3	3
+2	3	13	6.5000	6	7	2	2
+2	NULL	13	6.5000	6	7	2	2
+NULL	1	2	2.0000	2	2	1	1
+NULL	NULL	1	1.0000	1	1	1	2
+NULL	NULL	28	4.0000	1	7	7	8
```

Comment:
I am curious if this row has a proper avg given there are 8 values(one is NULL though) and their sum is 28. I expect this avg to be a decimal number.